js/dvActions.js

```diff
@@ -19,7 +19,7 @@ class DvActions {
                 ObsidianUtils.mutateFieldToDate({luxon, page, field: "done-date"});
             });
         if (start != null && end != null) {
-            actions = actions.where(p => p["do-date-obj"] >= start && p["do-date-obj"] <= end);
```

Comment:
`start` and `end` will be datetime objects which is why I was comparing to the `do-date-obj` field on the Action, it gets added by `ObsidianUtils.mutateFieldToDate`, which is honestly not that obvious reading the code now.
The `do-date` (and `done-date`) fields are Obsidian Link objects, which as far as I'm aware aren't comparable, or if they are I don't know what part of the link is compared with (the path to the file, or the display name of the link?)

What lead to this change?